roles/setup_gitops/tasks/local-registry.yml

```
@@ -0,0 +1,36 @@
+---
+- name: Assert that a path for the pull_secret is defined
+  ansible.builtin.assert:
+    that:
+      - sg_pullsecret_file | default('') | length > 0
```

Lets set `sg_pullsecret_file` to `''` in the defaults/main.yml so there is no need to define it in here either. 
```suggestion
      - sg_pullsecret_file | length > 0
```